src/app/test-record/defects/defects-edit/defects-edit.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DefectsEditComponent } from './defects-edit.component';
+
+describe('DefectsEditComponent', () => {
+  let component: DefectsEditComponent;
+  let fixture: ComponentFixture<DefectsEditComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ DefectsEditComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(DefectsEditComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
need a better description and snapshot for it please

Reply:
The defects edit section will be handled in future ticket. There is nothing in the template at the moment.